antlr-kotlin-target/src/main/resources/org/antlr/v4/tool/templates/codegen/Kotlin/Kotlin.stg

```diff
@@ -461,15 +463,13 @@ enterOuterAlt(_localctx, <currentOuterMostAltCodeBlock.alt.altNum>)
 >>
 
 CodeBlockForAlt(currentAltCodeBlock, locals, preamble, ops) ::= <<
-if (true) {
-    <locals; separator="\n">
-    <preamble; separator="\n">
-    <ops; separator="\n">
-}
+<locals; separator="\n">
```

Comment:
Was the if there for any specific reason? Like avoiding shadowing variables?

Reply:
It goes back to the Java template. The local scope is present in the Java template, and missing in some others.

With all the testing I've done I've never encountered a case where a shadowing happens.

Reply:
Makes sense